code/game/machinery/Sleeper.dm

```diff
@@ -397,11 +402,13 @@
 	. = TRUE //Returning TRUE means we successfully began the wake-up cycle. We will return immediately as the spawn() begins, not at the end.
 	on = TRUE
 	process()
-	var/sleeptime = min(5 SECONDS, 4*max(occupant.sleeping, occupant.paralysis))
+
 	spawn(sleeptime)
 		if(!src || !on) //the !src check is redundant from the nature of spawn() if I understand correctly, but better be safe than sorry
 			return
 		if(occupant)
+			if(occupant.reagents.has_any_reagents(list(STOXIN, STOXIN2, VALERENIC_ACID, CHLORALHYDRATE)))
```

Comment:
This check is not necessary, `remove_reagents` will work correctly without it. Remove it.

Reply:
I thought so but I saw other instances of this check in other places of code. Better safe than sorry when it comes to sanity.